client/extensions/woocommerce/app/order/order-details/row-shipping-refund.js

```diff
@@ -27,6 +27,7 @@ class OrderShippingRefundRow extends Component {
 			shippingTotal,
 			translate
 		} = this.props;
+		const total = Math.round( shippingTotal * 100 ) / 100;
```

Comment:
This is OK for US + CA, but WooCommerce supports different # of decimal places than 2 for currencies like ISK (Icelandic Krona) that don't use fractions. Maybe add a TODO here to use the site setting eventually?

Reply:
I've updated the code to use the localized `numberFormat` with the order's currency settings, so this shouldn't be a problem in the future.